README.md

```diff
@@ -42,84 +43,6 @@ The Inferno and CQF Ruler containers will be able to communicate, as they are on
 For STU3, the base URL is `http://<host>:8080/cqf-ruler-dstu3/`.
 For R4, the base URL is `http://<host>:8080/cqf-ruler-r4/`.
 
-### Preloading CQF Ruler
```

Comment:
Will this stuff be moved to another branch or repo? It is still important for us to have this stuff.

Reply:
It was moved to the wiki